stage1_fly/aci/aci-manifest.in

```diff
@@ -28,6 +28,14 @@
         {
             "name": "coreos.com/rkt/stage1/gc",
             "value": "/gc"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/stop",
+            "value": "/stop_fly"
+        },
+        {
+            "name": "coreos.com/rkt/stage1/interface-version",
+            "value": "3"
```

Comment:
With interface-version = 3, `/run` is supposed to understand `--hostname` according to [the "run" entrypoint doc](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md#arguments-added-in-interface-version-2).

Since fly is always running in the same UTS namespace as the host, fly should not change the hostname. So I think we should parse the flag `--hostname`  in stage1_fly/run/main.go and return a proper error message when it is given.